lib/sgtn-client/api/translation.rb

```diff
@@ -15,6 +15,9 @@ def self.getString(component, key, locale)
         str = getTranslation(component, key, locale)
         if str.nil?
           str = SgtnClient::Source.getSource(component, key, SgtnClient::Config.configurations.default)
+          if str.nil?
+            SgtnClient.logger.error "Can't find the key " + key + " in source path!"
```

Comment:
Suggest add single quote around key, log component if possible. So we know the necessary easily.